.eslintignore

```diff
@@ -1,24 +0,0 @@
-*.min.js
```

Comment:
We currently have some .min.js files in projects/packages/lazy-images/ that were being ignored by this, but aren't anymore because you moved the file. We probably want a projects/packages/lazy-images/.eslintignore to re-ignore them.

Reply:
moved file back to root